src/Indicator.vala

```diff
@@ -255,64 +255,35 @@ public class Sound.Indicator : Wingpanel.Indicator {
 
     public override Gtk.Widget? get_widget () {
         if (main_grid == null) {
-            int position = 0;
-            main_grid = new Gtk.Grid ();
-
             mpris = new Widgets.MprisWidget ();
 
-            mpris.close.connect (() => {
-                close ();
-            });
-            volume_control.notify["headphone-plugged"].connect (() => {
-                if (!volume_control.headphone_plugged)
-                    mpris.pause_all ();
-            });
-
-            main_grid.attach (mpris, 0, position++, 1, 1);
-
-            if (mpris.get_children ().length () > 0) {
-                var first_separator = new Wingpanel.Widgets.Separator ();
-
-                main_grid.attach (first_separator, 0, position++, 1, 1);
-            }
-
             volume_scale.margin_start = 6;
```

Comment:
Tiny thing, but I think we also want to move this line into `constructor` while we're here, because we don't change this value anywhere